### ISSUE TITLE
[Bug]: 使用Tableau连接MatrixOne，连上之后，只能显示表列表，但表字段显示无可用字段

### ISSUE THREAD

**Issue by xuanyiLee** [matrixorigin/matrixone#11284]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```Markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93):v0.8.0
- Hardware parameters:
- OS type:
- Others:
```

### Actual Behavior

使用Tableau连接MatrixOne，连上之后，只能显示表列表，但表字段显示无可用字段
![微信图片_20230815153544](https://github.com/matrixorigin/matrixone/assets/89380523/f703c31e-8b84-420c-af86-0371f4217806)

**Comment by YANGGMM** [comment]
"SELECT A.REFERENCED_TABLE_SCHEMA AS PKTABLE_CAT,NULL AS PKTABLE_SCHEM,A.REFERENCED_TABLE_NAME AS PKTABLE_NAME,A.REFERENCED_COLUMN_NAME AS PKCOLUMN_NAME,A.TABLE_SCHEMA AS FKTABLE_CAT, NULL AS FKTABLE_SCHEM,A.TABLE_NAME AS FKTABLE_NAME,A.COLUMN_NAME AS FKCOLUMN_NAME,A.ORDINAL_POSITION AS KEY_SEQ,CASE WHEN R.UPDATE_RULE = 'CASCADE' THEN 0 WHEN R.UPDATE_RULE = 'SET NULL' THEN 2 WHEN R.UPDATE_RULE = 'SET DEFAULT' THEN 4 WHEN R.UPDATE_RULE = 'SET RESTRICT' THEN 1 WHEN R.UPDATE_RULE = 'SET NO ACTION' THEN 3 ELSE 3 END AS UPDATE_RULE,CASE WHEN R.DELETE_RULE = 'CASCADE' THEN 0 WHEN R.DELETE_RULE = 'SET NULL' THEN 2 WHEN R.DELETE_RULE = 'SET DEFAULT' THEN 4 WHEN R.DELETE_RULE = 'SET RESTRICT' THEN 1 WHEN R.DELETE_RULE = 'SET NO ACTION' THEN 3 ELSE 3 END AS DELETE_RULE,A.CONSTRAINT_NAME AS FK_NAME,'PRIMARY' AS PK_NAME,7 AS DEFERRABILITY FROM INFORMATION_SCHEMA.KEY_COLUMN_USAGE A JOIN INFORMATION_SCHEMA.KEY_COLUMN_USAGE D ON (D.TABLE_SCHEMA=A.REFERENCED_TABLE_SCHEMA AND D.TABLE_NAME=A.REFERENCED_TABLE_NAME AND D.COLUMN_NAME=A.REFERENCED_COLUMN_NAME) JOIN INFORMATION_SCHEMA.REFERENTIAL_CONSTRAINTS R ON (R.CONSTRAINT_NAME = A.CONSTRAINT_NAME AND R.TABLE_NAME = A.TABLE_NAME AND R.CONSTRAINT_SCHEMA = A.TABLE_SCHEMA) WHERE D.CONSTRAINT_NAME IS NOT NULL  AND A.TABLE_SCHEMA = 'mo_catalog' AND A.TABLE_NAME = 'mo_account'  ORDER BY FKTABLE_CAT, FKTABLE_NAME, KEY_SEQ, PKTABLE_NAME, FKCOLUMN_NAME"

**Comment by YANGGMM** [comment]
ERROR 1064 (HY000): SQL parser error: table "referential_constraints" does not exist

**Comment by qingxinhome** [comment]
This issue requires the design of a metadata table function for foreign key constraints

**Comment by qingxinhome** [comment]
![image](https://github.com/matrixorigin/matrixone/assets/70939751/6c5be1f3-3845-4a01-af65-0c3442c415ac)

**Comment by qingxinhome** [comment]
The system view has been implemented, please verify and test it @arjunsk @dengn @aronchanisme @daviszhen @sukki37 @aressu1985

**Comment by aronchanisme** [comment]
准备Tableau工具中

**Comment by aronchanisme** [comment]
Fixed in `1.1-dev` and `main`.

Tableau Desktop: 2024.1.1 (2024.1.24.0308.0326)
<img width="1916" alt="image" src="https://github.com/matrixorigin/matrixone/assets/46433151/9bade0cb-aa99-4c80-ba6e-04135cee6637">

MO version: 
1. `1.1-dev`(`b48c930eaf363909f62433894855bf836108683c`)
<img width="1920" alt="image" src="https://github.com/matrixorigin/matrixone/assets/46433151/586166b4-143f-4d74-9a7b-342641f5f631">
<img width="1911" alt="image" src="https://github.com/matrixorigin/matrixone/assets/46433151/54826d7f-fdc9-450d-976c-62104fafc191">


2. `main`(`dc23dd174a336a248a1d3544494ac1b3a31c40b9`)
<img width="1920" alt="image" src="https://github.com/matrixorigin/matrixone/assets/46433151/465bb3f7-5b72-44ea-a074-4e4005600b88">

3. MySQL `8.0.36`
<img width="1886" alt="image" src="https://github.com/matrixorigin/matrixone/assets/46433151/5f7a6a5c-056f-4003-9844-2a2878db92a6">

**Comment by aronchanisme** [comment]
Hi @xuanyiLee  , please kindly take a look.
